stl/inc/__msvc_int128.hpp

```diff
@@ -860,7 +895,7 @@ struct _Int128 : _Uint128 {
         return _Left;
     }
 
-    _NODISCARD friend constexpr _Int128 operator>>(const _Int128& _Left, const _Int128& _Right) noexcept {
+    _NODISCARD_FRIEND constexpr _Int128 operator>>(const _Int128& _Left, const _Int128& _Right) noexcept {

```

Comment:
Above, `operator<<()` was changed to take `(const _Int128& _Left, const _Uint128& _Right)`. Was that change spurious, or should the same change be made here?

Reply:
This should be consistent